### ISSUE TITLE
Support for Default values

### ISSUE THREAD

**Issue by Shukla-Ankur** [zendesk/maxwell#1484]
MySql version: mysql  Ver 8.0.19, version 8.0.18
maxwell version: 1.26.1

All queries with Default value for any type of column fails
1.  Default in JSON column
```
CREATE TABLE testTable18 ( created_time BIGINT, updated_time BIGINT, tenant_id VARCHAR(255) NOT NULL,global_context_id VARCHAR(255) NOT NULL, auth_id VARCHAR(64) NOT NULL, version BIGINT ,pid BIGINT NOT NULL,command JSON NOT NULL DEFAULT (JSON_QUOTE("{'parent':'sched'}")),cpu VARCHAR(255) NOT NULL DEFAULT ('a1 b1'),PRIMARY KEY(`pid`) , Unique cpu (cpu(255),tenant_id) , INDEX cpu1 (cpu(255)));
```

```
 ERROR MysqlParserListener - (generic_type JSON (column_options (nullability NOT NULL)) (column_options (default_value DEFAULT ( (literal_with_weirdo_multistring (byte_literal JSON_QUOTE)))) ( "{'parent':'sched'}")
15:47:11,429 ERROR SchemaChange - Error parsing SQL: 'CREATE TABLE testTable18 ( created_time BIGINT, updated_time BIGINT, tenant_id VARCHAR(255) NOT NULL,global_context_id VARCHAR(255) NOT NULL, auth_id VARCHAR(64) NOT NULL, version BIGINT ,pid BIGINT NOT NULL,command JSON NOT NULL DEFAULT (JSON_QUOTE("{'parent':'sched'}")),cpu VARCHAR(255) NOT NULL DEFAULT ('a1 b1'),PRIMARY KEY(`pid`) , Unique cpu (cpu(255),tenant_id) , INDEX cpu1 (cpu(255)))'
15:47:11,429 ERROR AbstractSchemaStore - Error on bin log position Position[BinlogPosition[binlog.000022:153471], lastHeartbeat=1589451428532]
com.zendesk.maxwell.schema.ddl.MaxwellSQLSyntaxError: (
	at com.zendesk.maxwell.schema.ddl.MysqlParserListener.visitErrorNode(MysqlParserListener.java:93)
	at org.antlr.v4.runtime.tree.ParseTreeWalker.walk(ParseTreeWalker.java:41)
	at org.antlr.v4.runtime.tree.ParseTreeWalker.walk(ParseTreeWalker.java:52)
	at org.antlr.v4.runtime.tree.ParseTreeWalker.walk(ParseTreeWalker.java:52)
	at org.antlr.v4.runtime.tree.ParseTreeWalker.walk(ParseTreeWalker.java:52)
	at org.antlr.v4.runtime.tree.ParseTreeWalker.walk(ParseTreeWalker.java:52)
	at org.antlr.v4.runtime.tree.ParseTreeWalker.walk(ParseTreeWalker.java:52)
```

2. Default in int
```
CREATE TABLE testTable19 ( created_time BIGINT, updated_time BIGINT, tenant_id VARCHAR(255) NOT NULL,global_context_id VARCHAR(255) NOT NULL, auth_id VARCHAR(64) NOT NULL, version BIGINT ,pid BIGINT NOT NULL DEFAULT(1),command JSON NOT NULL DEFAULT (JSON_QUOTE("{'parent':'sched'}")),cpu VARCHAR(255) NOT NULL DEFAULT ('a1 b1'),PRIMARY KEY(`pid`) , Unique cpu (cpu(255),tenant_id) , INDEX cpu1 (cpu(255)));
```

```
ERROR MysqlParserListener - (default_value DEFAULT ( 1)
15:57:38,882 ERROR SchemaChange - Error parsing SQL: 'CREATE TABLE testTable19 ( created_time BIGINT, updated_time BIGINT, tenant_id VARCHAR(255) NOT NULL,global_context_id VARCHAR(255) NOT NULL, auth_id VARCHAR(64) NOT NULL, version BIGINT ,pid BIGINT NOT NULL DEFAULT(1),command JSON NOT NULL DEFAULT (JSON_QUOTE("{'parent':'sched'}")),cpu VARCHAR(255) NOT NULL DEFAULT ('a1 b1'),PRIMARY KEY(`pid`) , Unique cpu (cpu(255),tenant_id) , INDEX cpu1 (cpu(255)))'
15:57:38,882 ERROR AbstractSchemaStore - Error on bin log position Position[BinlogPosition[binlog.000022:197705], lastHeartbeat=1589452049321]
com.zendesk.maxwell.schema.ddl.MaxwellSQLSyntaxError: (
	at com.zendesk.maxwell.schema.ddl.MysqlParserListener.visitErrorNode(MysqlParserListener.java:93)
	at org.antlr.v4.runtime.tree.ParseTreeWalker.walk(ParseTreeWalker.java:41)
	at org.antlr.v4.runtime.tree.ParseTreeWalker.walk(ParseTreeWalker.java:52)
	at org.antlr.v4.runtime.tree.ParseTreeWalker.walk(ParseTreeWalker.java:52)
	at org.antlr.v4.runtime.tree.ParseTreeWalker.walk(ParseTreeWalker.java:52)
```

3. Deafult in varchar
```
CREATE TABLE testTable21 ( created_time BIGINT, updated_time BIGINT, tenant_id VARCHAR(255) NOT NULL,global_context_id VARCHAR(255) NOT NULL, auth_id VARCHAR(64) NOT NULL DEFAULT('abc'), version BIGINT ,pid BIGINT NOT NULL,command JSON NOT NULL,cpu VARCHAR(255) NOT NULL DEFAULT ('a1 b1'),PRIMARY KEY(`pid`) , Unique cpu (cpu(255),tenant_id) , INDEX cpu1 (cpu(255)));`
```

```
ERROR MysqlParserListener - (default_value DEFAULT ( 'abc')
15:59:19,886 ERROR SchemaChange - Error parsing SQL: 'CREATE TABLE testTable21 ( created_time BIGINT, updated_time BIGINT, tenant_id VARCHAR(255) NOT NULL,global_context_id VARCHAR(255) NOT NULL, auth_id VARCHAR(64) NOT NULL DEFAULT('abc'), version BIGINT ,pid BIGINT NOT NULL,command JSON NOT NULL,cpu VARCHAR(255) NOT NULL DEFAULT ('a1 b1'),PRIMARY KEY(`pid`) , Unique cpu (cpu(255),tenant_id) , INDEX cpu1 (cpu(255)))'
15:59:19,886 ERROR AbstractSchemaStore - Error on bin log position Position[BinlogPosition[binlog.000022:242406], lastHeartbeat=1589452156245]
com.zendesk.maxwell.schema.ddl.MaxwellSQLSyntaxError: (
	at com.zendesk.maxwell.schema.ddl.MysqlParserListener.visitErrorNode(MysqlParserListener.java:93)
	at org.antlr.v4.runtime.tree.ParseTreeWalker.walk(ParseTreeWalker.java:41)
	at org.antlr.v4.runtime.tree.ParseTreeWalker.walk(ParseTreeWalker.java:52)
	at org.antlr.v4.runtime.tree.ParseTreeWalker.walk(ParseTreeWalker.java:52)
	at org.antlr.v4.runtime.tree.ParseTreeWalker.walk(ParseTreeWalker.java:52)
	at org.antlr.v4.runtime.tree.ParseTreeWalker.walk(ParseTreeWalker.java:52)
	at org.antlr.v4.runtime.tree.ParseTreeWalker.walk(ParseTreeWalker.java:52)
	at org.antlr.v4.runtime.tree.ParseTreeWalker.walk(ParseTreeWalker.java:52)
```

**Comment by osheroff** [comment]
ok, mysql 8 changed some rules and now default column values get to be functions & whatever, enclosed in parens.  PR opened to fix.

**Comment by osheroff** [comment]
fixed in 1.26.2
